tools/generator/dfg/sam/sam_device_tree.py

```diff
@@ -268,7 +276,6 @@ def driverOrder(e):
                     inst = driver.addChild("instance")
                     inst.setValue(i[len(dtype):])
             if name == "gclk":
-                instance_pattern = re.compile(r"^(?P<per>([a-z]+))(?P<instance>([0-9]+))$")
                 driver.addSortKey(lambda e: (e.name, int(e["value"])))
```

Comment:
This key is incomplete though. There are several child node that have the same name and value in the gclk node.

Reply:
Let's see if that fixed it.